plugins/gadget.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gadget
+spec:
+  version: v0.1.0
+  homepage: https://github.com/kinvolk/inspektor-gadget
+  shortDescription: Collection of gadgets for debugging and introspecting Kubernetes applications
```

Comment:
no need for "Kubernetes" here.

If you can bring this description to 50-60 chars it would have a better chance of fitting to the screen without being truncated.

Reply:
I removed "Kubernetes" and made it shorter. Thanks for the review!